python/ray/autoscaler/updater.py

```diff
@@ -232,8 +242,10 @@ def run(self,
             timeout=120,
             exit_on_fail=False,
             port_forward=None,
-            with_output=False):
-
+            with_output=False,
+            ssh_options=None):
+        if not ssh_options:
+            ssh_options = self.default_ssh_options
```

Comment:
NIT: ssh_options = ssh_options or self.default_ssh_options

Reply:
Oh! That's awesome! I'll start using this more. It's much cleaner than 
```
ssh_options = ssh_options if ssh_options else self.default_ssh_options
```